lib/container/ContainerEntryDependency.js

```diff
@@ -0,0 +1,17 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Tobias Koppers @sokra, Zackary Jackson @ScriptedAlchemy, Marais Rossouw @maraisr
+*/
+
+"use strict";
+
+const Dependency = require("../Dependency");
+
+module.exports = class ContainerEntryDependency extends Dependency {
+	constructor(dependencies, name) {
+		super();
+		this.exposedDependencies = dependencies;
```

Comment:
> I think it would be nicer if the logic to create the ContainerExposedDependencies would be in ContainerEntryModule.build instead.

For this this dependency only need to store the `exposes` list.